lib/platform-stats.js

```diff
@@ -0,0 +1,18 @@
+import createDebug from 'debug'
+
+const debug = createDebug('spark:platform-stats')
+
+/**
+ * @param {import('pg').Client} pgClient
+ * @param {import('./preprocess').Measurement[]} honestMeasurements
+ */
+export const updateDailyNodeMetrics = async (pgClient, honestMeasurements) => {
+  debug('Updating daily node metrics, count=%s', honestMeasurements.length)
+  for (const m of honestMeasurements) {
+    await pgClient.query(`
+      INSERT INTO daily_node_metrics (station_id, metric_date)
+      VALUES ($1, now()::date)
```

Comment:
> So from the user's perspective, whether one measurement makes it into a certain day or not (if it's on the cusp of a day) is not based on anything they can verify.

Do you know if this is important? We're not talking about payouts here, only public stats. And if it doesn't make it into the current day, it will for sure make it into the next day. Yes this logic could be tighter, but I wouldn't worry about optimizing this now.

Reply:
I agree that you should not worry about this right now. 

We use `now()` in other places in public-stats too.

https://github.com/filecoin-station/spark-evaluate/blob/93b4d5ab7d013512870fcf974309f55c04089faf/lib/public-stats.js#L49-L52

BTW, I think PG will convert `now()` from date-time to date automatically, no need to do the conversion explicitly.

```suggestion
      VALUES ($1, now())
```